src/test/java/com/amadeus/ordering/transferOrders/transfers/CancellationIT.java

```diff
@@ -0,0 +1,76 @@
+package com.amadeus.ordering.transferOrders.transfers;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.aResponse;
+import static com.github.tomakehurst.wiremock.client.WireMock.post;
+import static com.github.tomakehurst.wiremock.client.WireMock.urlEqualTo;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import com.amadeus.Amadeus;
+import com.amadeus.Params;
+import com.amadeus.exceptions.ResponseException;
+import com.amadeus.resources.TransferCancellation;
+import com.github.tomakehurst.wiremock.WireMockServer;
+import java.io.IOException;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+// API at https://developers.amadeus.com/self-service/category/cars-and-transfers/api-doc/transfer-management
+public class CancellationIT {
+
+  WireMockServer wireMockServer;
+
+  private Amadeus amadeus;
+
+  /**
+   * Authentication is conducted for each test.
+   */
+  @BeforeEach
+  public void setup() {
+    wireMockServer = new WireMockServer(8080);
+    wireMockServer.start();
+
+    // API at https://developers.amadeus.com/self-service/apis-docs/guides/authorization-262
+    String address = "/v1/security/oauth2/token"
```

Comment:
As future improvement, we should try to centralize all the duplicated code (like this) in a class somewhere, so to reduce code to maintain and risk for bugs